src/content/blog/en/weekly-lt-challenge-journey.md

```
@@ -0,0 +1,73 @@
+// Expanding Community Activities
+const communityActivities = [
+  'Tech meetup presentations',
+  'Online community contributions',
+  'Mentoring junior developers',
+  'Open source contributions'
+];
+```
+
+### Q4: Transformation into a New Self
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)